dsp-meta-cmd/src/main-server.rs

```diff
@@ -46,8 +46,6 @@ fn main() {
 }
 
 async fn init_server() {
-    info!("Ivan was here!");
-
```

Comment:
comment (jocular): I was hoping to be the one removing this line. I would have replaced it with `info!("Removing graffiti... done.");` 😄

Reply:
hmm, don't know if I like this ...